notifications_utils/version.py

```diff
@@ -5,4 +5,4 @@
 # - `make version-minor` for new features
 # - `make version-patch` for bug fixes
 
-__version__ = "95.2.0"  # e423953f9dec74010897c62aa4e035de
+__version__ = "95.2.1"  # cca28798f43ed1eba5cb4b4a4b68797f
```

Comment:
If you’re changing the name of an argument this is a breaking change

Reply:
Done ☺️